recipes/aurora/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "aurora" %}
+{% set version = "0.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aurora-{{ version }}.tar.gz
+  sha256: a5df3b4a3c8d88fc6f3ea3c691dfe68128282edc5a6cb470914a55808ca739b6
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >=3.5
+  run:
+    - matplotlib-base
+    - numba
+    - numpy
+    - obspy
+    - pandas
+    - python >=3.5
+    - scipy
+    - xarray
+
+test:
+  imports:
+    - aurora
+    - aurora.signal
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/simpeg/aurora
+  summary: Processing Codes for Magnetotelluric Data
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - kkappler
```

Comment:
Can you verify either that you have spoken to the listed co-maintainers offline or have them comment on this PR that they are ok with being listed?

Reply:
I'm good with changes.

Reply:
Hi, Karl here - yes I'm happy to be a maintainer!